About page

nandini7637 opened the pull request:
# Related Issuse or bug
  - Info #35 

Fixes: #abo

# Proposed Changes
 - Info about changes Goes here
 
# Additional Info
  - Anything related Issues 
  
# Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot **|

AnkushSinghGandhi commented:
Hello, @nandini7637 I'm not sure what you're attempting to improve here, you just built another about page "about1.html" instead of revising "about.html," and I'm not sure what improvements you're proposing here. In my opinion, the "about" page is cleaner than the "about1" page that you created. However, I am closing this PR while simultaneously labeling it as "hacktoberfest-accepted" so that it will be counted towards hactoberfest.

AnkushSinghGandhi reviewed (commented):
Hello, @nandini7637 I'm not sure what you're attempting to improve here, you just built another about page "about1.html" instead of revising "about.html," and I'm not sure what improvements you're proposing here. In my opinion, the "about" page is cleaner than the "about1" page that you created. However, I am closing this PR while simultaneously labeling it as "hacktoberfest-accepted" so that it will be counted towards hactoberfest.

AnkushSinghGandhi commented:
If I missed something, please reopen the PR. Keep your Open Source spirit high!

nandini7637 commented:
@AnkushSinghGandhi yeah I have made another page because unnati di told me to make a seperate page, that's why I didn't touch the existing code!

nandini7637 commented:
@AnkushSinghGandhi regarding the changes, I have added animated cards for plan, vision, mission. Added Caraousal to show event highlights of Codexcrypt
Added a timeline for showing future or past events and shown the changed to @unnati914 di and I believe the page looks much better than previous one!

AnkushSinghGandhi commented:
> @AnkushSinghGandhi yeah I have made another page because @unnati914 di told me to make a seperate page, that's why I didn't touch the existing code!

Oh well I am not aware about that

AnkushSinghGandhi commented:
> @AnkushSinghGandhi regarding the changes, I have added animated cards for plan, vision, mission. Added Caraousal to show event highlights of Codexcrypt Added a timeline for showing future or past events and shown the changed to @unnati914 di and I believe the page looks much better than previous one!

Yeah it depends person to person so I am approving this PR @unnati914 😃😃
